python-sdk/src/astro/settings.py

```diff
@@ -36,3 +36,8 @@
 RAW_SQL_MAX_RESPONSE_SIZE = conf.getint(
     section="astro_sdk", key="run_raw_sql_response_size", fallback=-1
 )
+
+# Should Astro SDK automatically add inlets/outlets to take advantage of Airflow 2.4 Data-aware scheduling
+AUTO_ADD_INLETS_OUTLETS = conf.getboolean(
```

Comment:
@kaxil Can we add this to docs?

Reply:
Yeah @pankajkoti is going to do it as part of #852 as mentioned in the PR description